src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -280,6 +280,11 @@ private boolean setUpLogging(ContestConfig config) {
           Logger.log(Level.INFO, "Reading CDF cast vote record file: %s...", cvrPath);
           new CommonDataFormatReader(cvrPath, config).parseCvrFile(castVoteRecords);
           continue;
+        } else if (ContestConfig.getProvider(source) == Provider.CLEAR_BALLOT) {
```

Comment:
Did you merge in the latest in develop? I might be wrong, but it seems like @tarheel's latest changes may not have been captured. Please go ahead and do that if you haven't already.

Reply:
Just merged in #474, so that'll need to be accounted for.